src/engraving/property/propertyvalue.cpp

```diff
@@ -241,7 +241,9 @@ PropertyValue PropertyValue::fromQVariant(const QVariant& v, P_TYPE type)
 
     // other
     case P_TYPE::ACCIDENTAL_ROLE: return PropertyValue(Ms::AccidentalRole(v.toInt()));
+    case P_TYPE::SUB_STYLE:       return PropertyValue(v.toInt());
     default:
+        UNREACHABLE;
```

Comment:
If this is truely unreachable, because all possible values are dealt with, I'd rather remove the default branch and get a compiler warnign about an unhandled case as soon as another `P_TYPE` got added, but got forgotten here.

Reply:
But a log more have been added meanwhile it seems, so here's a merge conflict

Reply:
> If this is truely unreachable, because all possible values are dealt with, I'd rather remove the default branch and get a compiler warnign about an unhandled case as soon as another `P_TYPE` got added, but got forgotten here.

Fixed